metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -112,10 +171,10 @@ class GEProfilingConfig(ConfigModel):
     include_field_mean_value: bool = True
     include_field_median_value: bool = True
     include_field_stddev_value: bool = True
-    include_field_quantiles: bool = True
-    include_field_distinct_value_frequencies: bool = True
-    include_field_histogram: bool = True
-    include_field_sample_values: bool = True
+    include_field_quantiles: bool = False
+    include_field_distinct_value_frequencies: bool = False
+    include_field_histogram: bool = False
+    include_field_sample_values: bool = False
```

Comment:
It might be nicer to separate these defaults changes into a separate PR. If it's too much of a hassle, don't worry about it though!

Reply:
Thanks, I will create a separate pr from this.